client/src/pages/EditProfilePage/EditProfilePage.tsx

```diff
@@ -78,71 +227,267 @@ const EditProfilePage = () => {
+                  ? "bg-yellow-500"
+                  : "bg-blue-500"
+              }`}
+              onClick={() =>
+                setFormData((prevData) => ({
+                  ...prevData,
+                  availabilityForNetworking:
+                    !prevData.availabilityForNetworking,
+                }))
+              }
+            ></div>
+            <span className="ml-2 text-xs">
+              {formData.availabilityForNetworking ? 'Yes' : 'No'}
+            </span>
+          </label>
+
+          <label
+            className='block font-bold mb-2 text-sm'
+            htmlFor='linkedInProfile'
+          >
+            LinkedIn
+            <input
+              className='bg-gray-800 p-2 rounded text-white w-full'
+              id='linkedInProfile'
+              name='linkedInProfile'
+              type='text'
+              value={formData.linkedInProfile}
+              onChange={handleChange}
+            />
+          </label>
+          <label
+            className='block font-bold mb-2 text-sm'
+            htmlFor='gitHubProfile'
+          >
+            GitHub
+            <input
+              className='bg-gray-800 p-2 rounded text-white w-full'
+              id='gitHubProfile'
+              name='gitHubProfile'
+              type='text'
+              value={formData.gitHubProfile}
+              onChange={handleChange}
+            />
+          </label>
+          <label className='block font-bold mb-2 text-sm' htmlFor='twitter'>
+            Twitter
+            <input
+              className='bg-gray-800 p-2 rounded text-white w-full'
+              id='twitter'
+              name='twitter'
+              type='text'
+              value={formData.socialMediaLinks.twitter}
+              onChange={handleTwitterChange}
+            />
+          </label>
+          <label className='block font-bold mb-2 text-sm' htmlFor='blog'>
+            Blog
+            <input
+              className='bg-gray-800 p-2 rounded text-white w-full'
+              id='blog'
+              name='blog'
+              type='text'
+              value={formData.socialMediaLinks.blog}
+              onChange={handleBlogChange}
+            />
+          </label>
+          <label className='block font-bold mb-2 text-sm' htmlFor='skills'>
+            Skills
+            <input
+              className='bg-gray-800 p-2 rounded text-white w-full'
+              id='skills'
+              name='skills'
+              type='text'
+              value={skillInput}
+              onChange={handleSkillChange}
+              onKeyDown={handleSkillKeyDown}
+              placeholder='Type a skill and press Enter'
+            />
+          </label>
+          <div className='flex flex-wrap gap-2 mb-2 mt-4'>
+            {formData.skills.map((skill, index) => (
+              <div
+                key={index}
+                className='mb-2 mr-2 '
+                onClick={() => handleSpecialization(skill)}
```

Comment:
little UX bug - when clicking the X to remove a skill it removes the skill but also adds it as a specialization
![7A32F20F-3EDA-435F-A867-2939186F7542](https://github.com/Code-Hammers/code-hammers/assets/89617737/64b91073-8367-4182-bf9d-279e8c9b8415)

Reply:
Good catch. I was able to resolve this by stopping propagation when the remove button is hit.